### DISCUSSION TITLE
Background image for blocks / section

### DISCUSSION THREAD

**Discussion by ferenci-tamas** [google/docsy#2136] [Q&A] [answered]
Is it possible to add a background image to a `blocks / section` (instead of a simple coloring) the same way as it is done for `block / cover`?

**Comment by fekete-robert** [comment, marked as answer]
Not at the moment. You can either modify the shortcode to add support for a background image, or try some ugly hack in your content file like:
```
<div class="hack" >
{{< blocks/section color="transparent" type="features">}}

{{< /blocks/section >}}
</div>
```
And in your css:
```
.hack {
  background-image: url("/featured-background.jpg") !important;
}
```
And

**Reply by ferenci-tamas** [reply]
Thank you very much! I assume I must use this hack, no matter how ugly it is, if I want to avoid modifying docsy itself (which will also complicate future upgrades, if I am not mistaken).

**Comment by fekete-robert** [comment]
Probably you'd only have to copy the `themes/docsy/layouts/shortcodes/blocks/section.html` file into your project as `layouts/shortcodes/blocks/section.html` and modify it to fit your needs. This will override the default shortcode in docsy, so as long as a future version of this file doesn't include something that you want to use, you can keep using your own version and do the upgrades normally.

**Reply by ferenci-tamas** [reply]
OK, I see, thanks again!
